web/cats/static/cats/cat_profile_style.css

```diff
@@ -1,17 +1,54 @@
+*{
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+/*Style for a header*/
+header {
+  position: fixed;
+  top:0; left:0; right:0;
```

Comment:
Such formatting is a bit unusual. I'd recommend adding [csslint](https://github.com/pre-commit/mirrors-csslint) to your pre-commit hooks.